package.json

```diff
@@ -27,6 +27,28 @@
           "graphql/schema.graphql"
         ]
       }
+    },
+    {
+      "transpiler": "atom-ts-transpiler",
+      "glob": "{lib,test}/**/*.tsx",
+      "options": {
+        "compilerOptions": {
+          "module": "commonjs",
+          "moduleResolution": "node",
+          "target": "es6",
+          "noImplicitReturns": true,
+          "noFallthroughCasesInSwitch": true,
+          "noUnusedLocals": true,
+          "sourceMap": true,
+          "jsx": "react",
+          "strict": true
```

Comment:
I'm not sure if setting `"strict": true` will cause problems while we're mid-transition? Like, I think interfacing with JavaScript is easier with some of the strict checks enabled. I'm not sure about specifics, though.

Let's default to keeping it enabled until we hit problems though :smile: